src/halmos/__main__.py

```diff
@@ -566,6 +567,14 @@ def main() -> int:
                         if args.debug: traceback.print_exc()
                         continue
 
+                    if args.reset_bytecode:
+                        for assign in [x.split('=') for x in args.reset_bytecode.split(',')]:
+                            addr = con(int(assign[0].strip(), 0))
```

Comment:
not sure if i correctly understood your comment, but you can pass an address in hex too (or in binary or octal), like `--reset-bytecode '0x220f=0x....'` because of [base=0](https://docs.python.org/3/library/functions.html#int).